api/src/BcGov.Malt.Web/Models/SharePoint/SiteGroup.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace BcGov.Malt.Web.Models.SharePoint
+{
+         
+    public class SiteGroup
+    {
+        [JsonPropertyName("d")]
+        public SiteGroupData SiteGroupdata { get; set; }
+    }
+
+    public class SiteGroupData
+    {
+        [JsonPropertyName("__metadata")] 
+        public SiteGroupMetadata SiteGroupmetadata { get; set; }
+        [JsonPropertyName("Id")]
+        public int Id { get; set; }
+    }
+
+    public class SiteGroupMetadata
+    {
+        [JsonPropertyName("id")]
```

Comment:
not too sure how these things work so this may be a silly question Nancy but I am just wondering, in the `SiteGroupData` class we have the property named `Id` and then in `SiteGroup` it is called `id`
Is this intentional or how does it work with the case sensitivity?